Description: Replace the dependency on onetime with once.
Author: Paolo Greppi <paolo.greppi@libpf.com>
Forwarded: not-needed
Reviewed-By: Yadd <yadd@debian.org>
Last-Update: 2022-08-29

--- a/index.js
+++ b/index.js
@@ -1,8 +1,8 @@
 import process from 'node:process';
-import onetime from 'onetime';
+import once from 'once';
 import signalExit from 'signal-exit';
 
-const restoreCursor = onetime(() => {
+const restoreCursor = once(() => {
 	signalExit(() => {
 		process.stderr.write('\u001B[?25h');
 	}, {alwaysLast: true});
